### PR TITLE
Adds the services for backend router

### PR DESCRIPTION
**Pull request by nathan-lalli**
This adds the services to the front end for the new backend controllers

### CODE REVIEW

File: src/services/studentinstructor.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  create(data) {
+    return http.post("/performance-t3/studentinstructor", data);
+  }
+  update(id, data) {
+    return http.put(`/performance-t3/studentinstructor/${id}`, data);
+  }
+  getAll() {
+    return http.get("/performance-t3/studentinstructor");
+  }
+  getStudent(id) {
```
Comment (ethanimooney) [inline src/services/studentinstructor.js:12]: Is this getAllStudentInstructorsForStudentInfoId?

File: src/services/studentinstructor.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    return http.get(`/performance-t3/studentinstructor/studentinfoid/${id}`);
+  }
+  getInstructor(id) {
```
Comment (ethanimooney) [inline src/services/studentinstructor.js:15]: Same thing here, is this getAllStudentInstructorsForInstructorId?

**Review by ethanimooney — APPROVED** [review]
lgtm, just 2 questions

File: src/services/studentinstructor.js (hunk 1 above)
Comment (nathan-lalli) [reply src/services/studentinstructor.js:12]: This is just select * from StudentInstructors

Comment (ethanimooney) [reply src/services/studentinstructor.js:12]: No I mean getStudent

Comment (nathan-lalli) [reply src/services/studentinstructor.js:12]: Unless you are talking about getStudent(id)

That is get all instructors for that student ID

Comment (ethanimooney) [reply src/services/studentinstructor.js:12]: Cool

**Pull request merged**